src/components/graph-view.js

```diff
@@ -778,11 +790,13 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     if (shiftKey) {
       this.createNewEdge();
     } else {
-      const nodeMap = this.getNodeById(nodeId);
+      const nodeMapNode: INodeMapNode | null = this.getNodeById(nodeId);
+
+      if (nodeMapNode) {
+        const node: INode = nodeMapNode.node;
 
-      if (nodeMap) {
-        Object.assign(nodeMap.node, position);
-        onUpdateNode(nodeMap.node);
+        this.positionNodes(position, node, false);
```

Comment:
Curious why we needed to update the position here as node move might already have updated the postions

Reply:
The reason we update nodes here is because rendering nodes with `nodeMoving` false actually does a bunch of cleanup -- it removes a bunch of custom styling and attempts to clean up temporary elements. In the past, this code used to live under `handleNodeMove` but the previous change removed that to make node movement more stable.